src/preconnect.js

```diff
@@ -135,6 +144,20 @@ class Preconnect {
     return false;
   }
 
+  /** @private */
+  isPreloadSupported_() {
+    if (this.preloadSupported_ !== undefined) {
```

Comment:
If you cache the result in an instance var, then don't do this, just call it once and use the instance var after.